scripts/src/javatests/org/oppia/android/scripts/todo/TodoOpenCheckTest.kt

```diff
@@ -15,49 +20,46 @@ import java.io.PrintStream
 
 /** Tests for [TodoOpenCheck]. */
 class TodoOpenCheckTest {
-  private val outContent: ByteArrayOutputStream = ByteArrayOutputStream()
-  private val originalOut: PrintStream = System.out
-  private val TODO_CHECK_PASSED_OUTPUT_INDICATOR: String = "TODO CHECK PASSED"
-  private val TODO_SYNTAX_CHECK_FAILED_OUTPUT_INDICATOR: String = "TODO CHECK FAILED"
+  private companion object {
+    private const val TEST_AUTH_TOKEN = "abcdef1234567890"
+  }
+
+  private val outContent = ByteArrayOutputStream()
+  private val originalOut = System.out
+  private val TODO_CHECK_PASSED_OUTPUT_INDICATOR = "TODO CHECK PASSED"
+  private val TODO_SYNTAX_CHECK_FAILED_OUTPUT_INDICATOR = "TODO CHECK FAILED"
+  private val TODO_SYNTAX_CHECK_SKIPPED_OUTPUT_INDICATOR = "TODO CHECK SKIPPED"
   private val pathToProtoBinary = "scripts/assets/todo_exemptions.pb"
   private val wikiReferenceNote =
     "Refer to https://github.com/oppia/oppia-android/wiki/Static-Analysis-Checks" +
       "#todo-open-checks for more details on how to fix this."
+  private val regenerateNote =
+    "There were failures. Re-run the command with \"regenerate\" at the end to regenerate the " +
+      "exemption file with all failures as exempted."
```

Comment:
We could make this specific to "which" command, since we have a [script](https://github.com/oppia/oppia-android/wiki/Static-Analysis-Checks#how-to-run-static-checks-locally) for running all static checks locally, and the message could become ambiguous in this scenario.

Reply:
I'm not opposed to being clearer in this output--done.